python/xorbits/deploy/kubernetes/client.py

```diff
@@ -495,6 +524,8 @@ def stop(self, wait: bool = False, timeout: int = 0):
 
         api = CoreV1Api(self._api_client)
         api.delete_namespace(self._namespace)
+        if self._external_storage == "juicefs":
+            api.delete_persistent_volume("juicefs-pv")
```

Comment:
If use a static string here, when user starts multiple xorbits namespace in a same kubernetes cluster (with same juicefs csi), is there a data sharing which leads to insecure data case? Please verify this.